Dockerfile

```
@@ -1,26 +1,30 @@
+
+
 FROM python:3.12 AS build
 # Downgraded to python 3.12 so binary wheel for siphash24 is available
 
 WORKDIR /src
 
-RUN apt update && \
-apt install -y libclfft-dev git && \
-pip install build setuptools wheel cython
+RUN apt update && apt install -y ocl-icd-opencl-dev
+
+RUN pip install build
 
 COPY . .
 
 RUN python -m build --wheel
 
-RUN pip wheel -w dist --no-deps --no-use-pep517 gpyfft@git+https://github.com/geggo/gpyfft@v0.8.0
+# Build wheel for pyvkfft in build container to reduce final image size
+RUN pip wheel -w dist --no-deps pyvkfft
 
 FROM python:3.12-slim
 
-RUN apt update && apt install -y libclfft2
+# Adding pocl allows for opencl-accelerated fit on AMD CPUs
+RUN apt update && apt install -y ocl-icd-libopencl1 pocl-opencl-icd
```

```suggestion
RUN apt update && apt install -y ocl-icd-libopencl1
```

Let's look at pocl separately